CLAUDE.md

```
@@ -0,0 +1,107 @@
+To add a new boolean store:
+```ts
+import { createBooleanStore } from '$/store/bool.svelte'
+export const isMyThing = createBooleanStore()
+// read: isMyThing.value  |  write: isMyThing.set(true)
+```
+
+### Content
+
+Blog posts live in `src/content/blog/` as `.md`/`.mdx` files. Required frontmatter fields (enforced by Zod in `src/store/collections/blog.ts`):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)